evals/registry/solvers/whisper.yaml

```diff
@@ -0,0 +1,36 @@
+generation/whisper/gpt-4-turbo:
```

Comment:
I think it's not hard to understand, but it might make sense to comment with more info.

Something like: each audio message part is first processed by `whisper-1` model and then the messages are concatenated and sent to the solver below (e.g. GPT-4-Turbo).